frameworks/digital-outcomes-and-specialists-3/search_mappings/briefs.json

```diff
@@ -137,20 +137,29 @@
           "type": "keyword"
         },
         "dmtext_organisation": {
-          "type": "keyword"
+          "type": "text"
         },
         "dmtext_title": {
-          "type": "keyword"
+          "type": "text"
         },
         "dmtext_specialistRole": {
-          "type": "keyword"
+          "type": "text"
         },
         "dmfilter_specialistRole": {
           "type": "keyword",
           "normalizer": "filter_normalizer"
         },
         "dmtext_summary": {
-          "type": "keyword"
+          "type": "text",
+          "analyzer": "stemming_analyzer"
```

Comment:
In the G-Cloud mapping, `stemming_analyzer` is defined as an `analyzer`. Do we need to do it here too?

Reply:
Also looking at the G-Cloud mapping: is `"term_vector": "with_positions_offsets"` the magic sauce that does the search highlighting in the G Cloud `serviceDescription`?

Reply:
Damn it I didn't include those sections in the commit for some reason.